### PR TITLE
Operator Precedence - Add `print`, fix associativity of `instanceof` and unary operators

### PR DESCRIPTION
**Pull request by guilliamxavier**
See:
- https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4893#discussion_r398776216
- https://github.com/php/php-src/commit/2a04efe0e42c9d7ac150bc918e16edb96cf4f219 (part of)

### CODE REVIEW

File: language/operators.xml [hunk 1]
```diff
@@ -109,14 +109,14 @@
         </entry>
        </row>
        <row>
-        <entry>non-associative</entry>
+        <entry>left</entry>
```
Comment (cmb69) [inline language/operators.xml]: Shouldn't this be `(n/a)` as well?

Comment (guilliamxavier) [reply language/operators.xml]: `$a instanceof $b instanceof $c` is evaluated as
`($a instanceof $b) instanceof $c` (i.e. will always result in `false` but still valid syntax)
hence left-associative

Comment (cmb69) [reply language/operators.xml]: Ah, yes, you're right. Thanks!

**Comment by cmb69** [comment]
Applied as http://svn.php.net/viewvc?view=revision&revision=349661.

**Pull request closed without merging**